serverless/serverless-release-notes.adoc

```diff
@@ -8,17 +8,14 @@ toc::[]
 
 For an overview of {ServerlessProductName} functionality, see xref:../serverless/serverless-getting-started.adoc#serverless-getting-started[Getting started with OpenShift Serverless].
 
-include::modules/serverless-getting-support.adoc[leveloffset=+1]
-
 :FeatureName: Knative Eventing
 include::modules/technology-preview.adoc[leveloffset=+1]
 
 // Modules included, most to least recent
+// include::modules/serverless-rn-1-8-0.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-2.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-1.adoc[leveloffset=+1]
 include::modules/serverless-rn-1-7-0.adoc[leveloffset=+1]
-include::modules/serverless-rn-1-6-0.adoc[leveloffset=+1]
-include::modules/serverless-rn-1-5-0.adoc[leveloffset=+1]
 
 [id="additional-resources"]
```

Comment:
Not related this PR's changes really, but would recommend making this ID more unique, e.g., `serverless-release-notes-additional-resources`.

Reply:
Cool, I'll do that in another PR that applies for earlier versions too though if that's OK :slightly_smiling_face: adding it to my TODO list.